client/Makefile

```diff
@@ -15,8 +17,6 @@ GO_PACKAGES_REPO_PATH = $(addprefix $(repo_path)/,$(GO_PACKAGES))
 GOFMT = gofmt -e -l -s
 GOTEST = go test --cover --race -v
 
-VERSION := $(shell git describe --tags --abbrev=0 2>/dev/null)+$(shell git rev-parse --short HEAD)
```

Comment:
@bacongobbler or @mboersma would know better but client and API versioning is slightly different. Especially if we split it out into diff repos like planned